services/http/api/package.json

```diff
@@ -1,10 +1,12 @@
 {
   "name": "bb-api",
   "description": "Internal API for Bipolar Bridges Project",
-  "main": "main.js",
+  "main": "/dist/main.js",
   "scripts": {
-    "test": "jest tests",
-    "start": "node main.js",
+    "build": "rm -rf ./dist && tsc",
+    "build:mock": "cd tests/mocks/auth_server && yarn build",
+    "test": "jest dist/tests",
```

Comment:
we should look into using tsjest instead (see companion app repo for how to do this, specifically the testing workflow PR)

Reply:
Just added ts-jest!